tools/disasm_elf_msg.py

```diff
@@ -210,32 +210,32 @@ def disas_elfmsgs(start):
 
         if condition_type == 0:
             if elf_message_type == 0xE0 and b1 == 0 and not (b0 & 1):
-                print(f"ELF_MSG_END({ARG_1}),")
+                print(f"HINT_CMD_END({ARG_1}),")
             else:
-                print(f"ELF_MSG_FLAG({ARG_0}, {ARG_1}, {ARG_2}, 0x{b1:02X}), /* eventChkInf[{(b1 >> 4) & 0xF}] & 0x{1 << (b1 & 0xF):X} */")
+                print(f"HINT_CMD_FLAG({ARG_0}, {ARG_1}, {ARG_2}, 0x{b1:02X}), /* eventChkInf[{(b1 >> 4) & 0xF}] & 0x{1 << (b1 & 0xF):X} */")
             assert b3 == 0
         elif condition_type == 2:
-            print(f"ELF_MSG_DUNGEON_ITEM({ARG_0}, {ARG_1}, {ARG_2}, {item_ids[b1]}),")
+            print(f"HINT_CMD_DUNGEON_ITEM({ARG_0}, {ARG_1}, {ARG_2}, {item_ids[b1]}),")
             assert b3 == 0
         elif condition_type == 4:
-            print(f"ELF_MSG_ITEM({ARG_0}, {ARG_1}, {ARG_2}, {item_ids[b1]}, {item_ids[b3]}),")
+            print(f"HINT_CMD_ITEM({ARG_0}, {ARG_1}, {ARG_2}, {item_ids[b1]}, {item_ids[b3]}),")
```

Comment:
It looks like this script needs to be updated further since the argument order was changed in the macros

Reply:
I talked with tharo about this before opening the PR. think it would probably be easier for him to do it since he wrote it and knows how it works better. And it seems that even before my changes it got a little out of date in some regards.

So I would prefer it gets updated seperate from this PR, but if others want it changed now we can work toward making that happen